etc/prepare-tests.sh

```diff
@@ -4,12 +4,6 @@ cd "$(dirname "$0")"/.. || exit 1
 
 outputDir="build/sync-functions"
 
-mkdir -p "$outputDir"
-mkdir -p build/test-reports/
-
-# Set up JSHint configuration for the generated sync functions
-cp "etc/jshintrc-sync-function-template.json" "$outputDir/.jshintrc"
```

Comment:
These commands are completely superfluous: the `make-sync-functions` script automatically creates the output directory, the `npm run test-report` command creates the `build/test-reports` directory, and the JSHint configuration template file is copied again by a command later on in this very same file.